doc/classes/ProjectSettings.xml

```diff
@@ -681,6 +681,9 @@
 		<member name="debug/settings/stdout/verbose_stdout" type="bool" setter="" getter="" default="false">
 			Print more information to standard output when running. It displays information such as memory leaks, which scenes and resources are being loaded, etc. This can also be enabled using the [code]--verbose[/code] or [code]-v[/code] [url=$DOCS_URL/tutorials/editor/command_line_tutorial.html]command line argument[/url], even on an exported project. See also [method OS.is_stdout_verbose] and [method @GlobalScope.print_verbose].
 		</member>
+		<member name="debug/settings/ui/show_system_font_fallback_warnings" type="int" setter="" getter="" default="1">
+			If enabled, produces a warning when system font fallback is used.
+		</member>
```

Comment:
The enum setting can be replaced with just a boolean that defaults to `false`, that is overridden to `true` for the `editor_runtime` feature tag.